migrations/Migration20241023232123.ts

```diff
@@ -0,0 +1,11 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20241023232123 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('alter type "rollen_system_recht_enum" add value if not exists \'CRON_DURCHFUEHREN\';');
+
```

Comment:
`CRON_DURCHFUEREN` was already added in Migration20241010072717, they probably just forgot to use the migration-create command. It will still work this way, but I'd remove this.